Resources/Prototypes/SS220/Entities/Structures/barricade.yml

```
@@ -0,0 +1,423 @@
+#© SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+# Абстрактная базы
+- type: entity
+  id: EstBaseBarricade
+  description: A barricade made out of wood planks. It looks like it can take a few solid hits.
+  parent: BaseStructure
+  name: barricade
+  abstract: true
+  components:
+  - type: InteractionOutline
+  - type: Sprite
```

```suggestion
  - type: Sprite
```
перенеси путь к спрайту в базовые баррикады, и у наследуемых пиши
  - type: Sprite
    state: стейт спрайта